modules/apps/transfer/keeper/forward.go

```diff
@@ -0,0 +1,96 @@
+package keeper
```

Comment:
I love moving this to a separate file, btw.

Would naming this to `packet_forwarding` make it even easier to quickly grasp what it does? Maybe even just `forwarding` is also clearer - but I personally like packet_forwarding for clarity ^^

Reply:
I like `forwarding.go` more. I'll move forward with this, can change later/discuss if others disagree.